src/controllers/EthereumBlockchainController.ts

```diff
@@ -232,30 +257,14 @@ export class EthereumBlockchainController extends AbstractBlockchainController {
 		}
 	}
 
-	async getUserNonceMailer(userAddress: string) {
```

Comment:
Propose to remove that. We will abstract signature from client. WalletController will take care of preparing the sugnatire itself.